gated-infrastructure-deploy/gated-infrastructure-deploy.yml

```diff
@@ -75,6 +78,7 @@ jobs:
 
   - deployment: deployInfrastructure
     displayName: "Deploy Infrastructure"
+    container: ${{variables.terraformazurecontainer}}
```

Comment:
```
    container: ${{variables.terraformazurecontainer}}
```

`variables` isn't correct. There are no variables in this file. Only parameters.